modules/microshift-data-cleaning-certs.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-certs_{context}"]
+= Cleaning the certificates data
+
+Cleanup the {microshift-short} certificates by running the following script. This script is useful to reset {microshift-short} custom certificates so that they are recreated upon the {microshift-short} service restart. This script performs the following cleanup actions:
+
+* Stop and disable {microshift-short} services
+* Delete all {microshift-short} pods
+* Delete all {microshift-short} certificates
+
+.Procedure
+
+. Run the following command.
```

Comment:
```suggestion
. Execute the `microshift-cleanup-data` script, by running the following command.
```
We use the `jobs-to-done` goal plus phrasal verbs to tell users how to do every task.
See https://spaces.redhat.com/display/CCSENABLE/Phrasal+verbs

Reply:
done